shared/styles/style-guide.desktop.js

```diff
@@ -44,7 +44,8 @@ export const globalColors = {
 }
 
 export const globalResizing = {
-  login: {width: 700, height: 580},
+  // Add 25px to login for the OS menubar
+  login: {width: 700, height: 605},
```

Comment:
Marco's added commits to this branch that fix the window height and revert this height change now.  Thanks!